NikitaNechaiev/HW1/HomeAssigment1.py

```diff
@@ -0,0 +1,9 @@
+print("Hello, what is your name?")
```

Comment:
good job
look at this implementation:

name = input("What is your name?")
age = input("How old are you?")
city = input("Where do you live?")
print(f"Hello, {name}.\nYour age is {age}.\nYou live in {city}.")